ui/src/app/subnets/components/ReservedRangeForm/ReservedRangeForm.tsx

```diff
@@ -142,7 +142,7 @@ const ReservedRangeForm = ({
               disabled={showDynamicComment}
               label={Labels.Comment}
               name="comment"
-              placeholder="IP range comment (optional)"
+              placeholder="IP range purpose (optional)"
```

Comment:
Drive-by revert to the angular version.

@petermakowski sorry I might not have been very clear last night. I think it's hard in these cases to know whether this sort of inconsistency has crept in over time or if the column name and form label are deliberately different.

In this case "comment" is used in the CLI when creating ranges:
```
maas $PROFILE ipranges create type=dynamic \
    start_ip=$IP_DYNAMIC_RANGE_LOW end_ip=$IP_DYNAMIC_RANGE_HIGH \
    comment='This is a reserved dynamic range'
```
So the column could be labelled the same for consistency when users are switching between the two, but a designer could have discovered that when users are entering the comment in the web form that they're not sure what kind of comment to leave.

So this is the dilemma we face, we can ask our current designers what they think, but it's good to give them as much context as possible as they might not have the same insight as the original designer.